packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/translations/Messages_fr_FR.json

```diff
@@ -119,5 +119,24 @@
   "xdsl_details_modem_profile": "Profil du modem",
   "xdsl_details_modem_profile_orange": "Profil Orange",
   "xdsl_details_modem_profile_bouygues": "Profil Bouygues",
-  "xdsl_details_modem_profile_standard": "Profil Standard"
+  "xdsl_details_modem_profile_standard": "Profil Standard",
+  "xdsl_detail_mail_sending": "Envoi d'email via notre serveur SMTP",
+  "xdsl_detail_mail_sending_enable": "Activer",
+  "xdsl_detail_mail_sending_disable": "Désactiver",
+  "xdsl_detail_mail_sending_status_enabled": "Activé",
+  "xdsl_detail_mail_sending_status_disabled": "Désactivé",
+  "xdsl_detail_mail_sending_status_blockedForSpam": "Bloqué pour spams",
+  "xdsl_detail_mail_sending_blockedForSpam_description": "L'envoi de mail est actuellement bloqué suite à la détection d'une activité mail suspecte considérée comme \"spam\". Un mail avec les détails de ce blocage vous a été envoyé sur le mail rattaché à votre identifiant client. Si vous pensez que ce blocage est illégitime ou que vous avez corrigé un envoi de mail anormal de votre côté, merci de prendre contact avec notre support via un ticket depuis votre espace client ou en suivant <a href=\"https://help.ovhcloud.com/csm?id=csm_get_help\" target=\"_blank\">ce lien</a>",
```

Comment:
Maybe avoid to put html inside trad text 

(create separated <a> html with its trad)